### DISCUSSION TITLE
How to make linear animation?

### DISCUSSION THREAD

**Discussion by empz** [stefanwittwer/remotion-animated#9] [Q&A] [answered]
I'm doing a simple Move animation with the following config,
```tsx
<Animated
    style={{ left: 0 }}
    absolute
    out={30}
    animations={[
      Move({
        duration: 30,
        initialX: -100,
        initialY: 10,
        x: 0,
      }),
    ]}
  >
```

I want the element to move from -100 to 0 horizontally during 30 frames (which is the video's fps, so 1 second). The problem is that the defaults are using some kind of easing-out configuration, so by the frame 20 we're almost at x: 0. I need the element to reach x: 0 exactly at the frame 30.

I'm not familiar with Spring-based animations, but Remotion does have an [Easing.linear](https://www.remotion.dev/docs/easing#linear). Wouldn't be easier to be able to pass in one of this properties to the animation instead of having to mess with options like mass, damping, stiffness and overshootClamping???

Can someone tell me what mass, damping, stiffness and overshootClamping I need to use to get a fully linear animation?

Thanks

**Comment by stefanwittwer** [comment, marked as answer]
Hi there! Apologies for the late answer. Support for non-spring-animations has now finally landed in Remotion Animated 2.2.0 [(changelog)](https://github.com/stefanwittwer/remotion-animated/releases/tag/v.2.2.0), making this possible.

You can now achieve this by using the `Ease.Linear` easing behaviour in your animation:
```diff
Move({
  duration: 30,
  initialX: -100,
  initialY: 10,
  x: 0,
+ ease: Ease.Linear
})
```
You can learn more about [easing customisation on the docs page](https://www.remotion-animated.dev/docs/easing/custom).
Hope this helps!
